src/core/qgismobileapp.cpp

```diff
@@ -300,12 +301,13 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
     // import authentication method configurations
     for ( const QString &dataDir : dataDirs )
     {
-      QDir configurationsDir( QStringLiteral( "%1/auth/" ).arg( dataDir ) );
+      QDir configurationsDir( QStringLiteral( "%1/auth" ).arg( dataDir ) );
       if ( configurationsDir.exists() )
       {
         const QStringList configurations = configurationsDir.entryList( QStringList() << QStringLiteral( "*.xml" ) << QStringLiteral( "*.XML" ), QDir::Files );
         for ( const QString &configuration : configurations )
         {
+          qDebug() << QStringLiteral( "Importing authentication configuration file %1" ).arg( configurationsDir.absoluteFilePath( configuration ) );
```

Comment:
If you need to log, why don't you use the official `QgsLogger` API?

Reply:
@suricactus ,

QgsLogger -> shows up in the message log, not great.

QgsDebugMsg -> fails to print in the adb logcat

qDebug() -> :heart: 

It's not meant to end up merged, just helping out debugging an issue.

Reply:
If you plan to keep it, then change the include here `#include <qgslogger.h>`